lib/install/management/dispatcher.go

```diff
@@ -125,12 +131,60 @@ func (d *Dispatcher) initDiagnosticLogs(conf *metadata.VirtualContainerHostConfi
 	}
 }
 
+func (d *Dispatcher) registerExtension(conf *metadata.VirtualContainerHostConfigSpec) error {
+	log.Infoln("Registering VCH as a vSphere extension")
+	extensionManager := object.NewExtensionManager(d.session.Vim25())
+	extension := types.Extension{
+		Description: &types.Description{
+			Label:   "VIC",
+			Summary: "vSphere Integrated Containers Virtual Container Host",
+		},
+		Company: "VMware, Inc.",
+		Version: "0.2",
+		Key:     "com.vmware.vic",
+	}
+
+	var err error
+	var certbuffer, keybuffer bytes.Buffer
+	if certbuffer, keybuffer, err = certificate.CreateRawKeyPair(); err != nil {
```

Comment:
I would prefer to see the certificate is generated in cmd/vic-machine/create pkg, instead of here. There already has certificate generation for docker service.
This generation should happen before other VC resource creation, e.g. resource pool, bridge network, etc.